### DISCUSSION TITLE
How to fix the model params

### DISCUSSION THREAD

**Discussion by kd97** [apache/incubator-mxnet#14915] [Q&A]
After running the demo fine-tune.py,I watched the differences between my-model params file and pre-trained params file.It seemed all params have changed,I think the fine-tune way should fix some layers.So the params should almost be same. But the result showed it seemed not to work. Anyone can explain for me ? or I got a wrong result?

**Comment by vrakesh** [comment]
@kd97 Thank you for the question, Can you elaborate more on the use, case what kind of Model is this , and what is the use case. If it is possible. to provide a sample reproducible script , that would help as well

@mxnet-label-bot  add [Question]
